web/test/time/YouDo/Time/Parse/Test.hs

```diff
@@ -7,21 +11,41 @@ import Data.Time.Calendar.MonthDay
 import Data.Time.Clock
 import Data.Time.Format
 import Distribution.TestSuite
-import Distribution.TestSuite.QuickCheck
 import System.Locale (defaultTimeLocale)
 import Test.QuickCheck
 import Text.Printf
+import qualified Test.QuickCheck as Q
+import qualified Distribution.TestSuite as TS
 
 import YouDo.Time.Parse
 
+-- support for quick check without cabal-test-quickcheck
```

Comment:
@stebulus this stuff here is what I pulled out of that library. It should probably be put somewhere else at some time in the future if it's needed elsewhere.